### PR TITLE
Fix problem with full download and update links to https

### PR DESCRIPTION
**Pull request by zero-24**
This PR fix a problem with full download and update links to https

### CODE REVIEW

File: www/core/sts/extension_sts.xml [hunk 1]
```diff
@@ -32,7 +32,7 @@
 			<tag>stable</tag>
 		</tags>
 		<maintainer>Joomla! PLT</maintainer>
-		<maintainerurl>https://www.joomla.org</maintainerurl>
+		<maintainerurl>httpss://www.joomla.org</maintainerurl>
```
Comment (mbabker) [inline www/core/sts/extension_sts.xml]: Two s'

Comment (zero-24) [inline www/core/sts/extension_sts.xml]: fixed. Thanks.

**Comment by zero-24** [comment]
Any update here? Do we get a merge here or can this be closed?

BTW: this also fixes a "bug" with `fu ll` => `full` https://github.com/joomla/update.joomla.org/pull/5/files#diff-b7d110d9f243f5c7c8ee4bde4a066e13L195

**Comment by wilsonge** [comment]
I must push these live with the next release though.

**Comment by zero-24** [comment]
:+1:

**Comment by wilsonge** [comment]
Uploaded with Beta 3 tonight :)

**Comment by zero-24** [comment]
Great work @wilsonge !

**Pull request merged**